x-pack/plugins/apm/server/lib/helpers/get_document_sources.ts

```diff
@@ -44,64 +83,89 @@ export async function getDocumentSources({
         ? docTypeConfig.rollupIntervals
         : [RollupInterval.OneMinute]
     ).flatMap((rollupInterval) => {
-      const searchParams = {
-        apm: {
-          sources: [
+      return {
+        documentType,
+        rollupInterval,
+        meta: {
+          checkSummaryExistsField: false,
```

Comment:
Should this be `checkSummaryFieldExists`?

Reply:
Also: does it need to be inside `meta`? Further down `meta` is removed:

https://github.com/elastic/kibana/blob/cc6df776badef13005963127286842e2f596c817/x-pack/plugins/apm/server/lib/helpers/get_document_sources.ts#L174-L178